README.md

```diff
@@ -1 +1,50 @@
-# awesome-books
\ No newline at end of file
+# Awesome_books
+
+> This is my Awesome Book project.
+> I created this project by first designing it using wireframes.
+
+# Live View
+> To view this project live, please visit this [page]()
+
+# Getting Started
+
+> Visit my [repository](https://github.com/rdnrn/awesome-books/tree/awesome-books-app)
+> Clone the repository in your code editor using the URL above.
+> Change directory into the repository and make your desried changes.
+
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML- Hypertext Markup Language
+- CSS - Cascading Style Sheets
+- JS - Javascript
+
+## Authors
+
+1. Akanbi Dire
+
+- GitHub:@direakanbi
+- Twitter: @direakanbi
+- LinkedIn: @direakanbi
+
+## Author 2
```

Comment:
- [ ] OPTIONAL
Kindly remove this heading title `Author 2`, as there's already the heading title `Authors` above. This will make your ReadMe more profressional